testsuite/features/step_definitions/menu_steps.rb

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2010-2019 Novell, Inc.
+# Licensed under the terms of the MIT license.
+
+#
+# Menu links
+#
+
+# Menu link: 1st level
+When(/^I follow the left menu "([^"]*)"$/) do |arg1, arg2|
```

Comment:
arg2 is never used

Reply:
and please don't use generic `argN`, but explicit names like `menuN`

Reply:
Done. Good catch, thanks.